microsite/data/plugins/newrelic-dashboard.yaml

```
@@ -0,0 +1,15 @@
+---
+title: New Relic Dashboard
+author: '@mufaddal7'
+authorUrl: https://github.com/mufaddal7
+category: Monitoring
+description: Easily view your New Relic Dashboards in backstage , creates real-time snapshot of your dashboards
```

```suggestion
description: Easily view your New Relic Dashboards in Backstage, via real-time snapshots of your dashboards
```

Rewording is optional, but should get rid of the extra space :grin:

---

Done 👍
Thanks for the suggestion